firebase-ai-friendly-meals/android/app/build.gradle.kts

```diff
@@ -40,6 +40,7 @@ android {
     }
     buildFeatures {
         compose = true
+        buildConfig = true
```

Comment:
`buildConfig` was intentionally turned off by default starting in [AGP 8.0](https://developer.android.com/build/releases/past-releases/agp-8-0-0-release-notes#default-changes) in order to improve build performance.

It seems like the only place where we use that option in this project is to fetch the `APPLICATION_ID` for the FileProvider - can we just hardcode the applicationID there instead of getting from `BuildConfig` ?

Reply:
Done